src/Components/Aspire.Confluent.Kafka/ConfluentKafkaMetrics.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Concurrent;
+using System.Diagnostics.Metrics;
+
+namespace Aspire.Confluent.Kafka;
+
+internal sealed class ConfluentKafkaMetrics
+{
+    private readonly IMeterFactory _meterFactory;
+    private readonly Meter _meter;
+    private static readonly Dictionary<string, string> s_descriptions = new Dictionary<string, string>
```

Comment:
Do we really need this Dictionary? Can't these just be `const string`s? They are only used in 1 place.

Reply:
done